### DISCUSSION TITLE
Add option for "Twice a day"

### DISCUSSION THREAD

**Discussion by menisadi** [iSoron/uhabits#1643] [Feature Requests]
I want to use Loop for medication\pills tracking but there is not option to have "twice a day" habits (or in general more than once a day). 
This forces me to create a separate habit for each dose which makes the result cluttered and not easily interoperable. 
A simple option to select -X- times a day for every habit (with the option to choose reminders for every dose) will improve the experience for such usage.

**Comment by JOduMonT** [comment]
In fact, it would be nice to be able to choose unit like hours and minutes instead of just days
Such as an example I'm building an habit base on the 20-20-20 principles so I have an alert in my phone every 20 minutes during works hours.

**Comment by exanrauzer** [comment]
How the interface will look like?

**Reply by menisadi** [reply]
The same way there is "X times per week" there can be "X times per day". The main tracking frame can treat this as a measurable habit. So if I choose "twice a day" I will be able to set 2 reminders on which I could press "done" and then it will show this as a success for this day. Otherwise it can be shown as a miss or as a numerical value (50% or simply 1). The score will be computed in the same manner that "X times per week" is calculated (unless I didn't understand it correctly).

**Reply by exanrauzer** [reply]
Let's make it general to X times per day. And full daily input (2/2 3/3 ...) will give you check-marked day.

Feature would be more complex and combined: Default is that you are about to do the habit every day. But there is no reason for you to do the habit every day. You should be able to set fe: only 5 days a week. So computaion do not vary from how it is done right now, but there would be 2 setups available at once - you can do the habit X times per day and Y times per week or any other interval that is already in the app.

How it sounds?

**Comment by Mia0a-hi** [comment]
I created a task to select a reminder for every 1 day 2 times when the save fails, it automatically switches to every 2 days.
![0685d8d6bb79543ec83f2588a1ab321](https://github.com/user-attachments/assets/ef68c9d3-8f38-4898-bcbb-0a095f950c47)

**Comment by ZornHadNoChoice** [comment]
I want this too. I currently have a habit of brushing twice a day. I use a simple yes/no habit and just answer yes at the end of the day if I brushed twice.

This has many problem, e.g. it doesn't help with brushing at the right times, it doesn't remind me to brush and if I only brush once, I can only add a note to indicate that.

Another option is to implement multiple reminders and default increments to measurable habits. So I can tap the widget twice and each time it will increment by 1.
